qa/integration/services/service_container.rb

```diff
@@ -60,4 +60,24 @@ def stop_container
     @container.stop
     @container.delete(:force => true, :volumes => true)
   end
+
+  # Get environment variable from container's ENV.
+  def envvar(var)
+    # Exec command returns an array with [[stdout], [stderr], exit_code] layout.
+    res = @container.exec(['printenv', var])
+    res[0].join(" ").chomp
+  end
+
+  # Copy certificates from container to directory in the host.
+  def certificates(dest_dir)
```

Comment:
@wainersm ah actually there is (admittedly no obvious ones at all :D) reasons for doing that:

* we want to test logstash install without internet connection, this is currently done in a not so portable way => easy to script up with Docker
* Kafka is currently a pain outside CI. We have to run it from a fixed port because of the way its discovery works and it binds to a Socket in a lingering way => repeat runs on local fail if done in rapid succession :( (at least on my box)
* Cleanup for CTRL+C is a lot faster and cleaner than for a local Ruby process

All kinda small, but together it would be nice to have these fixed (imo) right? :)